docsrc/completions.rst

```diff
@@ -0,0 +1,74 @@
+Shell completions
+=================
+
+Argparse supports generating shell completion scripts for Bash, Zsh, and Fish.
```

Comment:
Maybe more active language?
> Argparse can generate shell completetions for Bash, Zsh and Fish.

Maybe add links for each shell? (not everyone who reads this may know what Fish is... the others would be mainly for consistency)